web/pages/GroupDetails.tsx

```
@@ -850,6 +893,26 @@ export const GroupDetails = () => {
                                     disabled={!isAdmin}
                                     required
                                 />
+                                <div className="space-y-1.5">
+                                    <label className={`text-sm font-bold ${style === THEMES.NEOBRUTALISM ? 'text-black uppercase' : 'opacity-70'}`}>
+                                        Group Currency
+                                    </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)